README.markdown

```diff
@@ -32,7 +32,7 @@ Installation
     $> make
     $> make install
 
-_Note: If you're compiling on Mac OS X, you'll need the 10.5 SDK installed. You can follow the instructions [here](http://hints.macworld.com/article.php?story=20110318050811544) to restore the 10.5 SDK to Xcode 4._
+_Note: If you're compiling on Mac OS X, you'll need the Xcode command-line tools installed._
```

Comment:
You need more than just the command-line tools installed, you also need the OS X SDK as the makefile links against it. (So you need Xcode installed.)

For support of all OS X platforms supported by 9.1 and 9.2, you need the 10.5 SDK. Granted, if you're just compiling it for yourself, you just need the SDK for your operating system.

I think this instruction should be left in as I don't want to drop support for 9.2